.github/SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+bancho.py is in a volatile state, therefore we only provide support for the latest version. It is the responsibility of the user to remain up-to-date.
```

Comment:
hmm i'm not sure about this, i generally try to provide support to all versions "within a reasonable timeframe", such as ~6 months - 1 year. if they have made modifications to their codebase, i ask that they provide their source code in a git (or alike) repository so that i can assess whether or not they've made any changes that affect what they're finding issues with.

Reply:
sure, i do think that we should not guarantee support for forks though as that is a complete nightmare. check to see if what i added works for you